Bottom: 9a22d550b1e7bbfe96fabd7ccd43da692d6776e4
Top:    294ad5db1a94b17e67167884f4010c642055c7b1
Author: Alexey Lyashkov <shadow@Alexeys-MacBook-Pro.local>
Date:   2016-01-14 11:34:16 +0300

Refresh of ib-sock-into.patch

---

diff --git a/ib-sock.c b/ib-sock.c
index 97fb5c6..8870570 100644
--- a/ib-sock.c
+++ b/ib-sock.c
@@ -9,11 +9,13 @@ static int cm_handler(struct rdma_cm_id *cmid, struct rdma_cm_event *event)
 		event->event, event->status, cmid->context, cmid);
 
 	switch (event->event) {
+	/* client related events */
 	case RDMA_CM_EVENT_ADDR_RESOLVED:
 	case RDMA_CM_EVENT_ROUTE_RESOLVED:
 	case RDMA_CM_EVENT_ESTABLISHED:
 	case RDMA_CM_EVENT_CONNECT_RESPONSE:
 		break;
+	/* server related events */
 	case RDMA_CM_EVENT_CONNECT_REQUEST:
 		break;
 	/* some common errors */
